pkg/osx/bitmask.pf.conf

```diff
@@ -0,0 +1,17 @@
+default_device = "en99"
```

Comment:
Is this `magic device` always? There is no different names for different ethernet/wifi devices?